.github/workflows/react-native-code-push-ci.yml

```diff
@@ -44,6 +55,8 @@ jobs:
         run: npm install
       - name: Install react-native-cli
         run: npm install react-native-cli
+      - name: Install reacti-native
+        run: npm install react-native
```

Comment:
And then install it here again

Reply:
It is iOS pipeline. There is a separate task to make if functional so you can ignore this part for now.